gigascience/gigadb-website
# switch datepicker format to yyy-mm-dd

# Pull request for issue: #1838 #1837

* change datepicker to use yyy-mm-dd format and show year and month dropdowns

## How to test?

* Go to any dtepicker and verify new format: http://gigadb.gigasciencejournal.com/adminFile/update/id/446
* Confirm presence of year and month selectors
* If possible verify in Safari

## How have functionalities been implemented?

- update datepicker description and placeholder
- update datepicker widget code to the new format
- Added year and month dropdowns to improve ux in safari as safari might not show arrows to change month

NOTE: this assumes that all datepickers on the site should adopt this format

## Any issues with implementation?

--

## Any changes to automated tests?

Updated a test that used the wrong date format
